Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -1286,6 +1316,59 @@ function vtkOpenGLRenderWindow(publicAPI, model) {
     }
     return modified;
   };
+
+  publicAPI.getGraphicsResourceForObject = (vtkObj) => {
+    if (!vtkObj) {
+      return null;
+    }
+    const vtko = model._graphicsResources.get(vtkObj);
+    const vtkh = model._graphicsResourceHash.get(vtkObj);
+    return { vtkObj: vtko, hash: vtkh };
+  };
+  publicAPI.setGraphicsResourceForObject = (vtkObj, gObj, hash) => {
+    if (!vtkObj) {
+      return;
+    }
+    model._graphicsResources.set(vtkObj, gObj);
+    model._graphicsResourceHash.set(vtkObj, hash);
+  };
+  publicAPI.getGraphicsMemoryInfo = () => {
+    let memUsed = 0;
+    model._graphicsResources.forEach((gObj, vtkObj) => {
+      memUsed += gObj.getAllocatedGPUMemoryInBytes();
+    });
+    return memUsed;
+  };
+  publicAPI.releaseGraphicsResourcesForObject = (vtkObj) => {
+    if (!vtkObj) {
+      return false;
+    }
+    model._graphicsResources.get(vtkObj)?.releaseGraphicsResources(publicAPI);
+    return (
+      model._graphicsResources.delete(vtkObj) &&
+      model._graphicsResourceHash.delete(vtkObj)
+    );
+  };
+  publicAPI.releaseGraphicsResources = () => {
```

Comment:
Should this be called in `delete()`? Or is that handled elsewhere?

Reply:
There is no `delete` at the OpenGL render window level because that is basically deleting the context.
This function should be called by applications that are using the switch data over time and need to free up the GPU allocations for old data.